flow-typed/npm/react-redux_v5.x.x.js

```diff
@@ -0,0 +1,98 @@
+// flow-typed signature: 8db7b853f57c51094bf0ab8b2650fd9c
```

Comment:
It's library definitions.
Added by https://github.com/flowtype/flow-typed

VSCode does the same for Typescript behind the scenes.
It's just saved in your computers cache instead of the projects VCS